src/pay/payment_producer.py

```diff
@@ -450,42 +555,67 @@ def try_to_pay(self, pymnt_cycle, rewards_type, network_config):
                 logger.warn(past_payment_state)
                 return True
 
-            # 1- get reward data
-            reward_model = self.reward_api.get_rewards_for_cycle_map(
-                pymnt_cycle, rewards_type
-            )
+            adjustments = {}
+            early_payout = False
+            current_cycle_rewards_type = rewards_type
+            # 1- adjust past cycle if necessary
+            if self.release_override == -11 and pymnt_cycle >= current_cycle:
```

Comment:
@nicolasochem this seems odd. I think it needs to work in the range of [-11,-6] ?

Reply:
@jdsika I have not tried in [-10, -6] range. I think we should limit the modes of opertion of TRD to 0, -5 and -11. I don't think anyone would use anything else.

With the current state of the code, you must use -11 for adjusted early payouts to kick in. Any other value in [-6, -10] will simply pay early payouts.

We should probably restrict the override value to 0, -5 and -11. I have not done it because these values would actually be different in testnets (because PRESERVED_CYCLE chain constant differs). Testnet is not in scope of the PR, but we should take a look at it otherwise.